R/surname_utils.R

```diff
@@ -248,12 +251,15 @@ predict_race_multi_barreled <- function(voter_file,
     surnames <- surnames[!(surnames %in% remove_patterns)]
   }
 
+  # Get merge_surnames function out from wru
+  merge_surnames_copy <- utils::getFromNamespace("merge_surnames", "wru")
+  
   # Use surname only
   if (surname_only) {
     new_voter_file <- data.frame(surname = surnames)
     # Calculate probabilities using surnames only
     probabilities <- suppressWarnings(
-      wru::merge_surnames(
+      merge_surnames_copy(
         voter.file = new_voter_file,
         surname.year = 2010,
```

Comment:
May wish to make this a parameter given the new data for 2020 is widely available.